website-howto.md

```diff
@@ -0,0 +1,19 @@
+# How to deploy this site
+
```

Comment:
Can you add some sentences on how to install yarn?
Could you also rename this file to CONTRIBUTION.md. Thanks.